addons/tutorial_01_get_started/manifest.json

```diff
@@ -26,39 +27,39 @@
         }],
         "next": {
           "op": "signal",
-          "qml_emitter": "serverListButton-btn",
+          "qml_emitter": "//serverListButton-btn",
           "signal": "visibleChanged"
         }
       },
       {
         "id": "countryAU",
-        "element": "serverCountryList/serverCountry-au",
+        "element": "//serverCountryList/{className=VPNServerCountry}[0]",
```

Comment:
Given this QT-Path will be evaluated, can this run the command part too? 
Is the scope sandboxed, or can any add-on theoretically change any property of any object?